### DISCUSSION TITLE
在vue组件里集成外部module的最佳实践是怎样的？如何利用vue开发module？感谢指导🤣

### DISCUSSION THREAD

**Discussion by lyric-zemin** [wangeditor-team/wangEditor#5479] [Q&A]
在vue组件里集成外部module的最佳实践是怎样的？如何利用vue开发module？感谢指导🤣

**Comment by moonlitusun** [comment]
module应该和vue没有什么关系，所以要看你是想实现什么样的功能。他的module一般都是为了增强解析或者显示标签的。
